src/components/Questions/Body.vue

```diff
@@ -298,9 +321,25 @@ export default defineComponent({
     }
 
     function preventKeypressIfApplicable(event: KeyboardEvent) {
-      // checks if character limit is reached in case it is set
-      if (!hasCharLimit.value) return;
-      if (!charactersLeft.value) event.preventDefault();
+      if (isQuestionTypeSubjective.value) {
+        // checks if character limit is reached in case it is set
+        if (!hasCharLimit.value) return;
+        if (!charactersLeft.value) event.preventDefault();
+      }
+      if (isQuestionTypeFloatNumerical.value) {
+        // console.log($event.keyCode); //keyCodes value
```

Comment:
Don't forget to remove the console statement.

Reply:
okay